### DISCUSSION TITLE
TypeError: text.replace is not a function

### DISCUSSION THREAD

**Discussion by xuedaobian** [meteorlxy/vue-showdown#32] [Q&A] [answered]
my repo addr : https://github.com/meteorlxy/vue-showdown/discussions
in Article.vue,i import md file as "import Idexmd from '../../public/server/index.md';",
then have the error : "TypeError: text.replace is not a function",
what can i do ? thank you

**Comment by meteorlxy** [comment, marked as answer]
see #12

**Reply by xuedaobian** [reply]
i can find the file
